assets/css/styles.css

```diff
@@ -0,0 +1,36 @@
+@import url('https://fonts.googleapis.com/css?family=Rubik:300,400&display=swap');
+
+body{ 
+    width: 100%; 
+    height: 100%; 
+    margin: 0px; 
+    padding: 0px; 
+    background: #E5E5E5;
```

Comment:
Nit: Ensure you are consistent with your CSS values. It is recommended for your CSS values to be in lowercase.